### PR TITLE
fix(curriculum): grid area template lesson

### LINKED ISSUE #38959: Lack of visual feedback with no explanation in grid-template-areas challenge
Forum thread: https://www.freecodecamp.org/forum/t/possible-bug-in-a-lesson/396043

The challenge is asking the camper to make changes to the `grid-template-areas` string, but it has no effect because no `grid-area` is defined.

I think at least the challenge should explain that it has no effect just yet but that the next challenge will show how to use the `grid-template-areas` by using the `grid-area` property.

Personally I'd like to change the focus of the challenge to simply be having the camper write the `grid-template-areas` string using the example code as a guide. But I realize this will require changes to not only the challenge text but also the test and video which is a lot of effort for such a small correction.

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #38959

### CODE REVIEW

**Review by gikf — DISMISSED** [review]
Looks okay to me :+1:

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
@moT01 The next lesson's solution and seed do not match this.

I realise this is not the same as the upcoming curriculum, but it does not make much sense mentioning the next step being a resolution to the topic.

**Comment by moT01** [comment]
Want to take another look when you get a chance @ShaunSHamilton? I tried to line it up with the next lesson better.

File: curriculum/challenges/english/01-responsive-web-design/css-grid/divide-the-grid-into-an-area-template.md [hunk 1]
```diff
@@ -15,25 +15,25 @@ You can group cells of your grid together into an <dfn>area</dfn> and give the a
 grid-template-areas:
   "header header header"
   "advert content content"
-  "footer footer footer";
+  "advert footer footer";
 ```
 
-The code above merges the top three cells together into an area named `header`, the bottom three cells into a `footer` area, and it makes two areas in the middle row; `advert` and `content`. **Note:** Every word in the code represents a cell and every pair of quotation marks represent a row. In addition to custom labels, you can use a period (`.`) to designate an empty cell in the grid.
+The code above groups the cells of the grid into four areas; `header`, `advert`, `content`, and `footer`. Every word represents a cell and every pair of quotation marks represent a row. Defining the areas won't have any visual effect right now. Later, you will make an item use an area to see how it works.
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/01-responsive-web-design/css-grid/divide-the-grid-into-an-area-template.md:21]:
```suggestion
The code above groups the cells of the grid into four areas; `header`, `advert`, `content`, and `footer`. Every word represents a cell and every pair of quotation marks represent a row.
```

File: curriculum/challenges/english/01-responsive-web-design/css-grid/divide-the-grid-into-an-area-template.md [hunk 2]
```diff
@@ -15,25 +15,25 @@ You can group cells of your grid together into an <dfn>area</dfn> and give the a
...
 
 # --instructions--
 
-Place the area template so that the cell labeled `advert` becomes an empty cell.
+Change the template so the `footer` area spans the entire bottom row.
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/01-responsive-web-design/css-grid/divide-the-grid-into-an-area-template.md:25]:
```suggestion
Change the template so the `footer` area spans the entire bottom row. Defining the areas won't have any visual effect right now. Later, you will make an item use an area to see how it works.
```

**Review by ShaunSHamilton — COMMENTED** [review]
This makes sense. Just one thing I would do:

**Review by ShaunSHamilton — APPROVED** [review]
Thanks for that back-and-forth, Tom. This LGTM 👍

**Review by gikf — APPROVED** [review]

**Pull request merged**